core/modules/qdisp/QueryResource.cc

```diff
@@ -75,7 +79,11 @@ void QueryResource::ProvisionDone(XrdSsiSession* s) { // Step 3
         _request = 0; // _session now has ownership
     } else {
         int code;
-        char const* msg = eInfo.Get(code);
+        char const* message = eInfo.Get(code);
+        std::string msg = "no message from XrdSsi, code may not be reliable";
+        if (message) {
+        	msg = message;
+        }
```

Comment:
Seeing as there are two identical defensive checks against a null ptr returned from eInfo.Get(code), do you think it makes sense to put the logic in an inline function?